### PR TITLE
Feature/readme

### PR DESCRIPTION
**Pull request by vladilyushchenko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,31 @@
...
+
+<b>Use existing docker-compose.yml file to run the server via terminal command</b>
+
+    docker-compose up
 
-Link to check Webhook info: 
+<b>Bot works via webhook technology. You should set webhook link via following link:</b>
 
-https://api.telegram.org/bot2102122174:AAEvhcaJAQ-cfThHhcfE6pbkKkwyFW9sVyc/getWebhookInfo
+    https://api.telegram.org/{bot_token}}/setWebhook?url={webhook-link}
```
Comment (zaplitny) [inline README.md:27]: {bot_token}} to {bot_token}

File: README.md [hunk 2]
```diff
@@ -1,11 +1,31 @@
...
+---
+1. Inline query (write in any chat "@fpm_blya_bot [...]" to see an option):\
+![alt text](https://i.ibb.co/VvtYxXG/cock-size-inline-query.png)
+2. Live chatting with bot in groups or bot's direct chat:\
+![alt_text](https://i.ibb.co/3W1Ggx3/chat-cock-size.png)
+
+### Install instruction
+
+---
 When starting an application set -Djava.net.useSystemProxies=true
```
Comment (zaplitny) [inline README.md:17]: If I understand well it's needed if we want to start application directly with java -jar and not related to docker-compose

File: README.md [hunk 3]
```diff
@@ -1,11 +1,31 @@
...
+    docker-compose up
 
-Link to check Webhook info: 
+<b>Bot works via webhook technology. You should set webhook link via following link:</b>
 
-https://api.telegram.org/bot2102122174:AAEvhcaJAQ-cfThHhcfE6pbkKkwyFW9sVyc/getWebhookInfo
+    https://api.telegram.org/{bot_token}}/setWebhook?url={webhook-link}
 
-Link to set new Webhook:
+You could get current state of bot via following link:
```
Comment (zaplitny) [inline README.md:29]: IMO, "You can get" is better

File: README.md [hunk 4]
```diff
@@ -1,11 +1,31 @@
...
-Telegram Api is unavailable in your country
+for JVM to setup proxy for application if
+Telegram Api is unavailable in your country.
+
+<b>Use existing docker-compose.yml file to run the server via terminal command</b>
+
+    docker-compose up
 
-Link to check Webhook info: 
+<b>Bot works via webhook technology. You should set webhook link via following link:</b>
```
Comment (zaplitny) [inline README.md:25]: "You should set" -> "Set" or "You need to set"
via THE following link and please give a link of what is a webhooks and so on. Like "You can read more about it at https://telegram-bla-bla"

File: README.md [hunk 5]
```diff
@@ -1,11 +1,31 @@
+# @fpm_blya_bot
+
+---
+@fpm_blya_bot is a bot for fun messaging in chats
+
+### Functional description
+
+---
+1. Inline query (write in any chat "@fpm_blya_bot [...]" to see an option):\
+![alt text](https://i.ibb.co/VvtYxXG/cock-size-inline-query.png)
```
Comment (zaplitny) [inline README.md:10]: IMO, image should not be stored on free external service. You can get out of limits

**Review by zaplitny — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (vladilyushchenko) [reply README.md:17]: Now I realized that this option is useless for me because now I run the bot via vpn

**Review by zaplitny — APPROVED** [review]

**Pull request merged**